src/utils/utils.test.tsx

```diff
@@ -177,3 +178,23 @@ describe("appUtils", () => {
     });
   });
 });
+
+describe("slugToString", () => {
```

Comment:
These look good. I don't know the extent of the titles of DC collections/divisions/items, but is there a need to escape and sanitize the URL before this function is called? This function _could_ sanitize the slug but it might be better a separate function. @avertrees maybe you might know if any titles include characters that should be escaped?